import ch.qos.logback.core.LogbackException;
public class FunctionUse528 {
public void funcUse() {
LogbackException logbackexception = new LogbackException();
}
}